app/src/androidTest/java/sdp/moneyrun/game/GameLobbyActivityInstrumentedTest.java

```diff
@@ -185,6 +202,28 @@ public void playersMissingUpdatesWithDB(){
     }
 
 
+    @Test
```

Comment:
How is the fact that the addPlayer fails checked in this test ?

Reply:
with the exception.except

Reply:
Oh sorry missed that